features/swap/domain/src/main/java/com/tangem/feature/swap/domain/SwapInteractorImpl.kt

```diff
@@ -1247,6 +1241,26 @@ internal class SwapInteractorImpl @Inject constructor(
         }
     }
 
+    /**
+     * Workaround to increase gas limit cause we calculate fee for random address
+     */
+    private fun Fee.increaseGasLimitBy(percentage: Int): Fee {
+        if (this !is Fee.Ethereum) return this
+        val gasLimit = this.gasLimit
+        val increasedGasPrice = this.amount.value?.movePointRight(this.amount.decimals)
+            ?.divide(gasLimit.toBigDecimal(), RoundingMode.HALF_UP)
+        val increaseGasLimit = gasLimit
```

Comment:
Вроде так должно быть
```suggestion
        val increasedGasLimit = gasLimit
```